### PR TITLE
Added create project button and new route

### LINKED ISSUE #38: Add 'Create Project' button

### PR DESCRIPTION
**Pull request by fmtabbara**
closes #38

### CODE REVIEW

File: client/src/app/components/CreateProject/styles.css [hunk 1]
```diff
@@ -40,20 +40,30 @@
...
 .form-wrapper button {
-  padding: 5px;
-  margin: 10px;
+  border: none;
+  height: 35px;
+  width: 75px;
+  padding: 8px;
+  margin: 2px;
+  background: #ec5252;
+  border: 1px solid #e8e9eb;
```
Comment (manu-4216) [inline client/src/app/components/CreateProject/styles.css]: You have on the first line border: none. Is that a mistake?

File: client/src/app/components/CreateProject/styles.css [hunk 2]
```diff
@@ -40,20 +40,30 @@
...
+  color: white;
+  border-radius: 3px;
+  font-size: 16px;
+  font-weight: bold;
+  cursor: pointer;
+  float: right;
```
Comment (manu-4216) [inline client/src/app/components/CreateProject/styles.css]: This is for aligning it to the right? I think that with float, the layout can be more difficult to reason about. And not sure, but the element dimensions that you declared above, will be ignored. 

For right positioning, there are other many alternatives. justify-text, position: relative, flex. These don't break the layout around the element.

File: client/src/app/components/CreateProject/styles.css [hunk 3]
```diff
@@ -40,20 +40,30 @@
...
+}
+
+.form-wrapper button:hover {
+  background: #b33b3b;
```
Comment (manu-4216) [inline client/src/app/components/CreateProject/styles.css:56]: Optional at this point, but I had introduced variables for colours, to try to reuse and share them across the app.

File: client/src/app/components/Header/HeaderRight/HeaderRight.js [hunk 4]
```diff
@@ -12,6 +12,9 @@ const HeaderRight = ({ direction }) => (
       <li>
         <Link to="/about-us">About Us</Link>
       </li>
+      <li className={styles.ctaButton}>
```
Comment (manu-4216) [inline client/src/app/components/Header/HeaderRight/HeaderRight.js:15]: Css-modules, cool!

File: client/src/app/components/Header/HeaderRight/HeaderRight.module.css [hunk 5]
```diff
@@ -22,6 +22,7 @@
 }
 .list a {
   text-decoration: none;
+  color: var(--dark-gray);
```
Comment (manu-4216) [inline client/src/app/components/Header/HeaderRight/HeaderRight.module.css:25]: I see that you used a colour variable. Cool!

File: client/src/app/components/CreateProject/styles.css (hunk 1 above)
Comment (fmtabbara) [reply client/src/app/components/CreateProject/styles.css]: Should be border: none. The other border property was the mistake!

File: client/src/app/components/CreateProject/styles.css (hunk 2 above)
Comment (fmtabbara) [reply client/src/app/components/CreateProject/styles.css]: Modified to flex

**Review by manu-4216 — APPROVED** [review]

**Pull request merged**